### DISCUSSION TITLE
Welcome to OpShin Discussions! Vote on features that you want to see implemented.

### DISCUSSION THREAD

**Discussion by nielstron** [OpShin/opshin#24] [Ideas / Improvements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jfbosch** [comment]
Hi all.
Jacques here from https://inversion.dev/cardano/
We build custom software systems (mostly .Net based) for our clients and have built a number of Cardano related solutions.
2 of us have completed Plutus Pioneers cohort 2, but have not built any Plutus smart contracts yet.
We are interested in seeing what might become possible with eopsin.

**Comment by nielstron** [comment]
Hi everyone,
Niels here. I have thorough experience with Cardano Smart Contracts from my involvement with building a DEX on Cardano. I have suffered through writing contracts in Plutus and want to improve the situation with eopsin!

**Comment by ghost** [comment]
Hi all, 

I'm Dave, I have experience with the Cardano CLI but I'm new to smart contracts. How would you suggest to get started?

**Reply by nielstron** [reply]
Hi Dave!
I would recommend you read through https://github.com/ImperatorLang/eopsin-example and https://github.com/ImperatorLang/eopsin ... and try to get your first `script.plutus` which you can use for building Smart Contract transactions with the CLI.

Don't miss this guide on how Smart Contracts work in general on Cardano: https://pycardano.readthedocs.io/en/latest/guides/plutus.html

**Reply by nielstron** [reply]
Oh and don't miss this gentle introduction into EUTxOs (or keep it in the back of your mind until you encounter "pubkeyhashes" and "redeemers": https://aiken-lang.org/fundamentals/eutxo

**Comment by sergi10** [comment]
HI all, here Sergi, I have a lot experience in Python and other lenguages. Following cardano technical ecosystem for two years.
Now I'm doing PPP fourth cohort and I completed the PPBL from gimbalabs, waiting for the next edition (https://gimbalabs.com/gimbalgrid/4).

**Reply by nielstron** [reply]
Hi @sergi10 

Since you are following the current PPP, are you also aware about the [EPP (eopsin-pioneer-program)?](https://github.com/OpShin/eopsin-pioneer-program) Are you considering to use it for self-study?

**Reply by sergi10** [reply]
Yes, I'm want to help OpShin to improve to be a great tool.

**Reply by nielstron** [reply]
Awesome! Drop into the discord server or ask questions here on the discussions page if you have any unclarities!
